content/documentation/sidebar.ts

```diff
@@ -33,6 +33,63 @@ export const sidebar: SidebarDefinition = [
     label: "RPC Reference",
     items: [
       "rpc/protocol",
+      {
+        label: "RPC Response Objects",
+        items: [
+          {
+            id: "rpc/objects/rpcAccountAssetBalanceDelta",
+            label: "Rpc Account Asset Balance Delta",
```

Comment:
I think the labels should match the object type exactly the way we do for the endpoint names:
<img width="232" alt="image" src="https://github.com/iron-fish/website/assets/57735705/26528ba8-3760-4cad-ae06-55c8f2d02ee2">

Reply:
Done!